doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -517,7 +517,10 @@ Libraries for networking
 
 * :ref:`lib_nrf_cloud` library:
 
-  * Updated to return negative :file:`errno.h` errors instead of positive ZCBOR errors.
+  * Updated:
+
+      * Return negative :file:`errno.h` errors instead of positive ZCBOR errors.
+      * CoAP download authentication no longer depends on coap_client.
```

Comment:
```suggestion
    * To return negative :file:`errno.h` errors instead of positive ZCBOR errors.
    * The CoAP download authentication to no longer depend on the :ref:`CoAP Client <zephyr:coap_client_interface>`.
```
I guess you are referring to Zephyr's CoAP Client library, right?

Reply:
changes are applied 👍🏻